src/jit/codegencommon.cpp

```diff
@@ -10853,6 +10853,139 @@ unsigned CodeGen::getFirstArgWithStackSlot()
 
 #endif // !LEGACY_BACKEND && (_TARGET_XARCH_ || _TARGET_ARM64_)
 
+void CodeGen::genSinglePush()
+{
+    genStackLevel += sizeof(void*);
+}
+
+void CodeGen::genSinglePop()
```

Comment:
From the name one thinks it will also generate code.  But it just adjusts stack level.  May be these should be renamed to genAdjustStackLevelForSinglePush/Pop() to be indicative of their purpose/functionality.  I know these are pre-existing routines and you didn't introduce them.  I would leave it to your call.